app/routes/user-routes.js

```diff
@@ -26,7 +26,7 @@ function signUp(req, res) {
 
     emailController.sendWelcomeEmail(user.email, verificationHash);
 
-    return res.status(200).send({
+    return res.status(201).send({
```

Comment:
> HTTP Status Code 201: The request has been fulfilled and has resulted in one or more new resources being created. 

`201` apt for instances where the user has uploaded file. I think `200` is good for the user creation